packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -9,68 +9,77 @@
           :cancel="closeReadmeContentModal"
           :space="space"
         ></readme-content-modal>
-        <div
-          class="oc-grid oc-px-m oc-mt-m"
-          :class="{ 'oc-child-width-1-1@s': imageExpanded, 'oc-child-width-1-3@s': !imageExpanded }"
-        >
-          <div v-if="imageContent">
-            <div class="oc-position-relative">
-              <img
-                :class="{ expanded: imageExpanded }"
-                class="space-overview-image oc-cursor-pointer"
-                alt=""
-                :src="'data:' + imageContent.mimeType + ';base64,' + imageContent.data"
-                @click="toggleImageExpanded"
-              />
-            </div>
+        <div class="oc-px-m oc-mt-m" :class="{ 'oc-flex': !imageExpanded }">
+          <div v-if="imageContent" :class="{ 'oc-width-1-4 oc-mr-l': !imageExpanded }">
+            <img
+              :class="{ expanded: imageExpanded }"
+              class="space-overview-image oc-cursor-pointer"
+              alt=""
+              :src="'data:' + imageContent.mimeType + ';base64,' + imageContent.data"
+              @click="toggleImageExpanded"
+            />
           </div>
-          <div>
-            <div class="oc-flex oc-mb-s oc-flex-middle">
-              <h1 class="space-overview-name oc-text-truncate">{{ space.name }}</h1>
+          <div :class="{ 'oc-width-3-4': !imageExpanded }">
+            <div class="oc-flex oc-mb-s oc-flex-middle oc-flex-between">
+              <div class="oc-flex oc-flex-middle">
+                <h1 class="space-overview-name oc-text-truncate">{{ space.name }}</h1>
+                <oc-button
```

Comment:
Could you add a little bit of margin to the left, so that the three dots button has a tiny bit more distance from the title?